src/Module.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-zendframework-bridge/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ZendFrameworkBridge;
+
+use Laminas\ModuleManager\Listener\ConfigMergerInterface;
+use Laminas\ModuleManager\ModuleEvent;
+use Laminas\ModuleManager\ModuleManager;
+
+class Module
+{
+    /**
+     * Initialize the module.
+     *
+     * Type-hinting deliberately omitted to allow unit testing
+     * without dependencies on packages that do not exist yet.
+     *
+     * @param ModuleManager $moduleManager
+     */
+    public function init($moduleManager)
```

Comment:
I believe we can have typehint here?

Reply:
I deliberately chose not to so I could test without having a dependency on a package that does not yet exist.

I'll add it once we have transferred the packages to their new homes.